test_twitter.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+
+from twitter import Twitter
+
+
+@pytest.fixture
+def local_picture():
+    with open("test_pic.jpg", "br") as f:
+        return f.read()
+
+
+@pytest.fixture
+def picture_id(local_picture):
+    twitter = Twitter()
+    pic_id, pic_name = twitter.upload_photo("test", local_picture)
+    return pic_id
+
+
+@pytest.fixture
+def tweet_id():
+    twitter = Twitter()
+    tw_id = twitter.create_post("test")
+    return tw_id
+
+
+def test_upload_photo_correct(local_picture):
+    twitter = Twitter()
+    returned_id, returned_name = twitter.upload_photo(name="Test_name", data=local_picture)
+    assert returned_id is not None
+    assert returned_name is not None
+    assert isinstance(returned_id, int)
+    assert isinstance(returned_name, str)
+    assert returned_id != ''
+    assert returned_name != ''
+
+
+def test_upload_photo_incorrect(local_picture):
+    twitter = Twitter()
+    not_picture = "string"
+    with pytest.raises(TypeError):
+        twitter.upload_photo(name="Test_name", data=not_picture)
```

Comment:
Only this line will execute, then exception will be raised and the next line will be skipped

Reply:
fixed